pkg/cmd/flags.go

```diff
@@ -20,3 +21,37 @@ func ValidateUpdateSliceFlags(cmd *cobra.Command, args []string) error {
 	}
 	return nil
 }
+
+const (
+	flagDoRunMigrations = "do-migrations"
+	flagSkipMigrations  = "skip-migrations"
+)
+
+func SetMigrationFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool("do-migrations", true, "Specify if you want to force migrations to run")
+	cmd.Flags().Bool("skip-migrations", false, "Specify if you want to skip migrations")
+}
+
+func ShouldRunMigrations(cmd *cobra.Command, happyConf *config.HappyConfig) (bool, error) {
+	if cmd.Flags().Changed(flagDoRunMigrations) && cmd.Flags().Changed(flagSkipMigrations) {
+		return false, errors.Errorf(
+			"Flags %s and %s cannot be specified at the same time",
+			flagDoRunMigrations,
+			flagSkipMigrations,
+		)
+	}
+
+	if cmd.Flags().Changed(flagDoRunMigrations) {
+		run, err := cmd.Flags().GetBool(flagDoRunMigrations)
+		return run, errors.Wrapf(err, "could not read flag %s", flagDoRunMigrations)
+	}
+
+	if cmd.Flags().Changed(flagSkipMigrations) {
+		skip, err := cmd.Flags().GetBool(flagSkipMigrations)
+		run := !skip
```

Comment:
Why have this copy here?

Reply:
because sometimes these double negatives are confusing so I like to be explicit and give it a descriptive name for the reader. Doesn't have to be there, but I've found it helps avoid some types of bugs